app/config/Configuration.scala

```diff
@@ -39,6 +40,7 @@ class Configuration {
 
   lazy val stepFunctionArn = StateMachineArn.fromString(config.getString("supportWorkers.arn")).get
 
-  implicit val settings = Settings.fromConfig(config.getConfig("switches"))
+  private val s3 = AwsS3Client.getClient
```

Comment:
My preference would be to inject the dependency on the AWS S3 client; which is marginally easier to do given `Configuration` is already a class.